# Author: Michael Lauer <mickey@vanille-media.de>
# Date:   Thu Jan 18 00:27:21 2007 +0000
# alsa-lib: fix "error: field 'tstamp' has incomplete type" when including alsa headers with strict compilation options
#
# Acquired from OpenEmbedded

Upstream-Status: Pending

Index: alsa-lib-1.0.13/include/global.h
===================================================================
--- alsa-lib-1.0.13.orig/include/global.h
+++ alsa-lib-1.0.13/include/global.h
@@ -29,6 +29,8 @@
 #define __ALSA_GLOBAL_H_
 
 /* for timeval and timespec */
+#define __need_timeval
+#define __need_timespec
 #include <time.h>
 
 #ifdef __cplusplus
